firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigSharedPrefsClientTest.java

```diff
@@ -349,4 +352,19 @@ public void clear_hasSetValues_clearsAll() {
     assertThat(info.getConfigSettings().getMinimumFetchIntervalInSeconds())
         .isEqualTo(DEFAULT_MINIMUM_FETCH_INTERVAL_IN_SECONDS);
   }
+
+  @Test
+  public void getCustomSignals_isNotSet_returnsEmptyMap() {
+    assertThat(metadataClient.getCustomSignals()).isEqualTo(Collections.emptyMap());
+  }
+
+  @Test
+  public void getCustomSignals_isSet_returnsCustomSignals() {
```

Comment:
Can we have more test cases for example: mix between long and string signals. Enter invalid signals. Since this is a change on public api we should really think about all the use cases user could potentially entry.

Reply:
Sure, I've added more comprehensive tests now.